### ISSUE TITLE
PS3Build format

### ISSUE THREAD

**Issue by humbertodias** [ps3dev/ps3libraries#66]
I saw a similar project [psp-packages](https://github.com/pspdev/psp-packages) that uses the format [PSPBUILD](https://github.com/pspdev/psp-packages/blob/master/sdl2-image/PSPBUILD) to organize and build packages.
How about if we created a PS3BUILD?
Implementing something like this we were able to package individually each package on [release](https://github.com/pspdev/psp-packages/releases/tag/master-812413259) section
![image](https://github.com/ps3dev/ps3libraries/assets/9255997/d5f072c9-a27a-4a20-8c54-95d6bb5e09da)

**Comment by bucanero** [comment]
Yes, I think it would be great to have ready-to-use binaries of these commonly used ps3 libraries.

**Comment by clienthax** [comment]
Looking at
